install/update_931_932.php

```diff
@@ -73,6 +73,12 @@ function update931to932() {
    $migration->addKey('glpi_states', 'is_visible_enclosure');
    $migration->addKey('glpi_states', 'is_visible_pdu');
 
+   // Create a dedicated token for rememberme process
+   if (!$DB->fieldExists('glpi_users', 'cookie_token')) {
+      $migration->addField('glpi_users', 'cookie_token', 'string', ['after' => 'api_token_date']);
+      $migration->addField('glpi_users', 'cookie_token_date', 'datetime', ['after' => 'cookie_token']);
+   }
+
```

Comment:
Either you add this in a new file, or you don't apply the fix.
This will never be run.

Reply:
This can be using the cli --force option

Reply:
Creating another file will create a mess for user who will try to update to GLPI 9.4.0 or 9.4.1 in a second time.
I would prefer to add a clear message on the release message to indicate that the update command have to be launched using CLI and maybe give DB queries to execute for people who cannot use CLI.

Reply:
DB change has been dropped